.taprc

```diff
@@ -0,0 +1,7 @@
+# vim: set filetype=yaml :
```

Comment:
I'm not opposed to the vim line, but please name the file `.taprc.yaml` and add `"tap": "./.taprc.yaml"` to `package.json`.

https://github.com/tapjs/tapjs/issues/1026

Reply:
I agree with you, but can I just remove it for this PR?

I think I added it because of a linting problem, or it could be the tap cli (more likely)

This seems like it could be done much faster at the `org` scale with a script

Cc @Fdawgs

Reply:
It's not something we need to go off and blow up all of our inboxes with. If you're doing some work in a repo and see it lacking, go ahead and update it. That's the intention here: you're already making changes regarding this file and how it is recognized by the tool.

Let me be clear: _please_ do not flood my inbox with changes across all of our repos for this.

Reply:
> Let me be clear: please do not flood my inbox with changes across all of our repos for this.

haha sure, yeah I get how annoying those are!